src/Goerli-DssSpell.t.sol

```diff
@@ -128,10 +92,16 @@ contract DssSpellTest is GoerliDssSpellTestBase {
             assertEq(spell.expiration(), spellValues.deployed_spell_created + spellValues.expiration_threshold, "TestError/spell-expiration");
 
             // If the spell is deployed compare the on-chain bytecode size with the generated bytecode size.
-            // extcodehash doesn't match, potentially because it's address-specific, avenue for further research.
+            // extcodehash doesn't match because the code of DssExec is different on every deployment since it takes a timestamp as a constructor param.
+            // However, the code hash of the spell action should match.
             address depl_spell = spellValues.deployed_spell;
             address code_spell = address(new DssSpell());
+
             assertEq(getExtcodesize(depl_spell), getExtcodesize(code_spell), "TestError/spell-codesize");
+
+            address depl_action = DssExecLike(depl_spell).action();
+            address code_action = DssExecLike(code_spell).action();
+            assertEq(getExtcodehash(depl_action), getExtcodehash(code_action), "TestError/action-codehash");
```

Comment:
at this point, I think we should remove this so it does not block this spell.  We can research further why we have issues with the action check that was ported from mainnet.

Reply:
I'm not seeing this specific error. Tests passing on my end.